crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1683,3 +1736,37 @@ impl ScheduleBuildSettings {
         }
     }
 }
+
+/// Event used to request changes to [`Schedule`]s
```

Comment:
Interesting. I don't know how I feel about this being very broad. If we add e.g. runtime system insertion does it belong in this enum?

Reply:
I tried to do something generic because I did see mentions of plans to eventually support run-time modification of `Schedule`.  I'm happy to rename it to something else like `SteppingEvent`.

Reply:
Yeah let's see what other reviewers say.